src/PProf.jl

```diff
@@ -81,6 +81,7 @@ overwriting the output file. `PProf.kill()` will kill the server.
   ignored/hidden through the web UI to be ignored from totals when computing percentages.
 """
 function pprof(data::Union{Nothing, Vector{UInt}} = nothing,
+               lidict::Union{Nothing, Dict} = nothing,
                period::Union{Nothing, UInt64} = nothing;
```

Comment:
After looking at this again, should `period` also be a kwarg, so that changes like this don't affect it?

@vchuravy - do you remember why we made it a positional argument?

Reply:
No that information is gone from my brain. Damn LRU caches

Reply:
K, i'm going to make it a keyword argument then, and rename it to like, `duration`?

Reply:
sounds good.

Reply:
Oh, jk, it's the "sampling period". K i'm going to rename it to `sampling_period`. Or maybe `sampling_delay`, since that matches `Profile.init(delay=0.01)`.

I like `sampling_delay`.

Reply:
Okay, I have updated this to a kwarg. Please take another look! :)